src/gap-skipper.js

```diff
@@ -0,0 +1,88 @@
+import Ranges from './ranges';
+
+let adaptiveSeeking;
+let gapSkipper;
+let seekingObject;
+
+gapSkipper = function(options) {
```

Comment:
this should probably be a const rather than being defined above, also I don't think it takes any options anymore. 
ex: `const gapSkipper = function() {`. The same goes for `adaptiveSeeking`